frontend/src/metabase/nav/components/ProfileLink.jsx

```diff
@@ -74,89 +70,100 @@ export default class ProfileLink extends Component {
         icon: null,
         link:
           isAdmin && MetabaseSettings.isPaidPlan()
-            ? "https://www.metabase.com/help-premium?utm_source=in-product&utm_medium=menu&utm_campaign=help"
-            : "https://www.metabase.com/help?utm_source=in-product&utm_medium=menu&utm_campaign=help",
+            ? `https://www.metabase.com/help-premium?utm_source=in-product&utm_medium=menu&utm_campaign=help&instance_version=${tag}&diag=${compactBugReportDetailsForUrl}`
+            : `https://www.metabase.com/help?utm_source=in-product&utm_medium=menu&utm_campaign=help&instance_version=${tag}`,
 
         externalLink: true,
         event: `Navbar;Profile Dropdown;About ${tag}`,
       },
       {
         title: t`About Metabase`,
         icon: null,
-        action: () => this.openModal("about"),
+        action: () => openModal("about"),
         event: `Navbar;Profile Dropdown;About ${tag}`,
       },
       {
         title: t`Sign out`,
         icon: null,
-        action: () => this.props.handleLogout(),
+        action: () => handleLogout(),
         event: `Navbar;Profile Dropdown;Logout`,
       },
     ].filter(Boolean);
   };
 
-  render() {
-    const { modalOpen } = this.state;
-    const { tag, date, ...versionExtra } = MetabaseSettings.get("version");
-    // don't show trademark if application name is whitelabeled
-    const showTrademark = t`Metabase` === "Metabase";
-    return (
-      <div>
-        <EntityMenu
-          tooltip={t`Settings`}
-          items={this.generateOptionsForUser()}
-          triggerIcon="gear"
-          triggerProps={{
-            color: color("text-medium"),
-            hover: {
-              backgroundColor: color("brand"),
-              color: color("text-white"),
-            },
-          }}
-        />
-        {modalOpen === "about" ? (
-          <Modal small onClose={this.closeModal}>
-            <div className="px4 pt4 pb2 text-centered relative">
-              <div className="text-brand pb2">
-                <LogoIcon height={48} />
-              </div>
-              <h2
-                style={{ fontSize: "1.75em" }}
-                className="text-dark"
-              >{t`Thanks for using Metabase!`}</h2>
-              <div className="pt2">
-                <h3 className="text-dark mb1">
-                  {t`You're on version`} {tag}
-                </h3>
-                <p className="text-medium text-bold">
-                  {t`Built on`} {date}
-                </p>
-                {!/^v\d+\.\d+\.\d+$/.test(tag) && (
-                  <div>
-                    {_.map(versionExtra, (value, key) => (
-                      <p key={key} className="text-medium text-bold">
-                        {capitalize(key)}: {value}
-                      </p>
-                    ))}
-                  </div>
-                )}
-              </div>
+  useEffect(() => {
```

Comment:
Instead of adding a `componentDidMount`, I think this is a perfect opportunity to convert this class component to a functional component as it isn't too complex.